sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -52,23 +53,36 @@
     AmqpMessageHeader,
     AmqpMessageProperties,
 )
+from ._pyamqp._message_backcompat import LegacyMessage, LegacyBatchMessage
+from ._pyamqp.message import Message
+from ._transport._pyamqp_transport import PyamqpTransport
 
-from ._pyamqp import constants, utils as pyutils
-from ._pyamqp.message import BatchMessage, Message
```

Comment:
We no longer need to import it here, since the BatchMessage is built inside the `amqp_transport.build_batch_message()` method called inside the EventDataBatch constructor.